src/shared/utils/pagination.util.ts

```diff
@@ -0,0 +1,22 @@
+import { PaginationMetadata } from '../constants/pagination.constant';
+import { PaginationValues } from '../types/pagination.type';
+
+export function getPaginationOffset(page: number, limit: number) {
+  return {
+    take: limit ?? PaginationMetadata.Limit,
```

Comment:
Unnecessary check. Now that limit is always defined, you don't have to have this check

Reply:
removed